Baekjoon/2720.py

```diff
@@ -0,0 +1,16 @@
+'''
+문제링크:https://www.acmicpc.net/problem/2720
+카테고리:그리디
+문제 해설: 거스름돈 각각 개수 구하기
+쿼터(Quarter, $0.25)의 개수, 다임(Dime, $0.10)의 개수, 니켈(Nickel, $0.05)의 개수, 페니(Penny, $0.01)
+'''
+t=int(input())
+
+for i in range(t):
+  a=int(input())
+  coins=[25,10,5,1]
+  count=[0,0,0,0]
+  for index, coin in enumerate(coins):
+    count[index] += a//coin
+    a%=coin
```

Comment:
```suggestion
  result=[]
  for coin in coins:
    result.append(a//coin)
    a%=coin
```

빈 배열에 결과값을 하나씩 추가하는 방식도 있겠습니다